include/seqan3/search/detail/search_scheme_algorithm.hpp

```diff
@@ -112,13 +112,13 @@ class search_scheme_algorithm : protected policies_t...
                                 search_param error_state,
                                 delegate_t const & on_hit_delegate)
     {
-        if constexpr (traits_t::search_best_hits || traits_t::search_all_best_hits || traits_t::search_strata_hits)
+        if constexpr (traits_t::search_single_best_hit || traits_t::search_all_best_hits || traits_t::search_strata_hits)
```

Comment:
```suggestion
        if constexpr (!traits_t::search_all_hits)
```

Otherwise: the line is too long